tests/resources/cleanup_test.go

```diff
@@ -0,0 +1,26 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/rancher/tfp-automation/tests/extensions/provisioning"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type CleanupTestSuite struct {
+	suite.Suite
+}
+
+func (r *CleanupTestSuite) TestCleanup() (bool, error) {
```

Comment:
the return looks funky. Error shouldn't be returned at all afaik, since we never return the error. But also, is this TestCleanup function called anywhere? If so, in what way? I would push to not call the test function directly, but perhaps a helper function that this TestCleanup can call, along with whatever else needs this result.

Reply:
This function specifically is cleaning up resources leftover in the `main.tf` when performing local testing. This comes in handy in those situations. For jenkins jobs, this is not explicitly called, that is correct, but I wouldn't want to flat out remove this.